results/results.json

```diff
@@ -0,0 +1 @@
+{"threshold": "0.50", "precision": "0.74", "recall": "0.41", "f1": "0.53"}
```

Comment:
Just a quick question, which part of the dvc.yaml are these results (and the ones below) linked to? Sorry you might have mentioned it earlier but now I can't figure out anymore what evaluation these are part of (vs. the results above)... they look good though, especially the wt ones 🙂

Reply:
so there's three pipelines in the yaml:
- the 'normal' xlinear one, which is the full MeSH tree, except for the c. 200 terms we are now excluding with `create_inclusion_list.py`. The evaluation results from this model are here in `results.json`
- the WT pipeline, which only uses the MeSH tags we use at Wellcome (c. 85% of the full MeSH tree), the results of which are in `results_wt.json`
- the toy pipline, which only uses a handful of MeSH tags, the results of which are in `results_toy.json`